packages/client/src/forms/configuration/formConfig/utils/defaultConfig.ts

```diff
@@ -62,8 +62,11 @@ export function defaultQuestionToConfigField(
       getGroup(question.identifiers, defaultForm),
       previewGroup
     ),
-    configFields: [configField]
+    configFields: [configField],
```

Comment:
Your prettier format-on-save doesn't seem to be working 🤔 Could you see it does as otherwise anyone else working on this file later on would change make automatic changes to the file just by saving it?